### PR TITLE
Chore/hdk 152 d

### LINKED ISSUE #57: HDK 152

### PR DESCRIPTION
**Pull request by dauphin3**
Fixes #57

### CODE REVIEW

**Comment by ocarissa** [comment]
@dauphin3 check out why your tests are failing! :)

**Comment by ocarissa** [comment]
@dauphin3 all your checks are passing now! @harlantwood can you do Zeek's code review, or if you have already, can you approve so he can merge. 👍

**Comment by ocarissa** [comment]
Further to this: @dauphin3, pay mind to if this PR gets out of date with main again! We did the rebase process successfully last Friday, but it's now Monday. You may need to re-do your rebase before merge. Keep those notes we created together, handy!

File: nix/sources.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (harlantwood) [inline nix/sources.json:1]: This whole nix dir should be nuked IMO

File: zomes/trust_atom/coordination/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hc_zome_tg_coordination"
+version = "0.0.10-dev"
+edition = "2021"
+rust-version = "1.63.0"
+
+[lib]
+name = "hc_zome_tg_coordination"
+# path = "src/lib.rs"
```
Comment (harlantwood) [inline zomes/trust_atom/coordination/Cargo.toml:9]: Remove?

**Comment by harlantwood** [comment]
Great work!  Couple small requests.

File: zomes/trust_atom/coordination/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hc_zome_tg_coordination"
```
Comment (harlantwood) [inline zomes/trust_atom/coordination/Cargo.toml:2]: I’m thinking a better name is “trust_atom” — looking at it from the perspective of the importing code.  

Maybe then the other one is “trust_atom_integrity”

Thoughts?

Comment (dauphin3) [reply zomes/trust_atom/coordination/Cargo.toml:2]: yea that sounds good

File: nix/sources.json (hunk 1 above)
Comment (dauphin3) [reply nix/sources.json:1]: this is auto-generated from niv, would you like to keep or remove from the default.nix?

(I had tried deleting and commiting but it looks like it showed after pushing rebase changes)

Comment (harlantwood) [reply nix/sources.json:1]: Please nuke and if still green that is better. I don’t think it was auto generated, I think I added it.

Comment (ocarissa) [reply nix/sources.json:1]: @harlantwood @dauphin3 

When I spoke with Zeek, he said he did nuke it, and then they popped back up. Zeek can you try that again to confirm your actions?

**Comment by harlantwood** [comment]
Nuke it harder? 😂
-- 
Harlan T Wood
https://core.network
It is my commitment to catalyze the creation and adoption of luminous tools
to enable free and open transformation of organizations, individuals, and
the world.  I dream of an age of enlightened creativity, and the end of
scarcity on planet Earth.

File: nix/sources.json (hunk 1 above)
Comment (dauphin3) [reply nix/sources.json]: got it this time, but checks not passing

**Comment by dauphin3** [comment]
all good now

**Comment by harlantwood** [comment]
Also want to have a clean coherent commit message before merging

 the current commit message is a bit of a mess:

IMG_8148


What we want is a coherent message that tells people what has changed. Format:
Summary
(Blank line)
bullet point 1
Bullet point 2
[12:53](https://corenexus-team.slack.com/archives/C046MM7QWCT/p1669200812686259)
Example  commit:

IMG_8149

**Comment by harlantwood** [comment]
Superseded by #59

**Pull request closed without merging**